tests/test_base_device.py

```diff
@@ -1166,7 +1170,6 @@ def test_process_estimator_job(self, measurements, expectation):
 
         for data in result.metadata:
             assert isinstance(data, dict)
-            assert list(data.keys()) == ["variance", "shots"]
```

Comment:
The differences are pretty significant. Technically I could hardcode it so that the formats match but that didn't seem like a good solution either so I deleted it for now. Let me know if you think there's some kind of better assertion here! Open to suggestions

Reply:
I don't think this was an important line of the test, I'm happy to just have it removed 😅